accounts/keystore/keystore_passphrase.go

```diff
@@ -90,6 +91,23 @@ func (ks keyStorePassphrase) GetKey(addr common.Address, filename, auth string)
 	return key, nil
 }
 
+func StoreKey(dir, auth string, scryptN, scryptP int) (error, common.Address) {
```

Comment:
dox pls :)

Reply:
And reverse the return parameters. Error should always be the last return.

Reply:
Please use 

```go
func StoreKey(dir, auth string, scryptN, scryptP int) (common.Address, error) {
    _, a, err := storeNewKey(&keyStorePassphrase{dir, scryptN, scryptP}, crand.Reader, passphrase)
   return a.Address, err
}
```